sdk/core/System.ServiceModel.Rest/api/System.ServiceModel.Rest.net461.cs

```diff
@@ -0,0 +1,62 @@
+namespace System.ServiceModel.Rest
```

Comment:
Just like in Azure.Core.dll, we should refactor types that users will be referring to from the types that are building blocks. The former should be in System.ServiceModel.Rest; the later in System.ServiceModel.Rest.Core

Reply:
Cool, will fix.  I'll assume these should parallel the `Azure` and `Azure.Core` namespace groupings, but let me know if not.

Reply:
Updated